Assignment_1/main.swift

```diff
@@ -0,0 +1,267 @@
+
+//
+//  main.swift
+//  assignment1
+//
+//  Created by Sanskar on 11/01/22.
+//
+import Foundation
+
+protocol Tax
+{
+    static func calculateTax(price : Double) -> Double
+}
+
+class RawTax : Tax
+{
+    static func calculateTax(price : Double) -> Double
+    {
+        return (0.125 * price)
+    }
+}
+
+class ManufacturedTax : Tax
+{
+    static func calculateTax(price : Double) -> Double
```

Comment:
Change logic here:
Tax is an attribute of an Item. Implementation using Item as an object is more scalable

Reply:
done